modules/uid2IdSystem.js

```diff
@@ -183,29 +71,24 @@ export const uid2IdSubmodule = {
    * @returns {uid2Id}
    */
   getId(config, consentData) {
-    const result = getIdImpl(config, consentData);
+    const mappedConfig = {
```

Comment:
if this isnt supposed to work in europe, should it kick out when gdpr applies?

Reply:
I've updated it to bail out if GDPR applies.